shell/platform/android/io/flutter/embedding/android/FlutterFragmentActivity.java

```diff
@@ -783,4 +785,10 @@ private boolean isDebuggable() {
   protected FrameLayout provideRootLayout(Context context) {
     return new FrameLayout(context);
   }
+
+  /** Returns the {@code #FRAGMENT_CONTAINER_ID} of the activity's content view. */
+  @NonNull
+  public int getFragmentContainerId() {
```

Comment:
Maybe, instead of exposing this method, we could expose the ID of this view, and use `ViewUtils.generateViewId` instead of the current value.

Here's an example of how `ViewUtils.generateViewId` is used in FlutterActivity:
https://github.com/flutter/engine/blob/279e3af0a1c8f19286dded8aabbfbf752c2a55fe/shell/platform/android/io/flutter/embedding/android/FlutterActivity.java#L218